source/uwp/Renderer/dll/dll.cpp

```diff
@@ -2,15 +2,14 @@
 //    Copyright (C) Microsoft.  All rights reserved.
 //
 #include "pch.h"
-#include <windows.h>
-#include <wrl.h>
-#include <wrl\wrappers\corewrappers.h>
+#include <wrl.H>
+#include <module.g.cpp>
 
 using namespace Microsoft::WRL;
 
 STDAPI DllGetActivationFactory(_In_ HSTRING activatableClassId, _COM_Outptr_ IActivationFactory** factory)
```

Comment:
Can you create a new cppwinrt project to see if it's still need wrl namespace?
I'm wondering if you can use winrt::IActivationFactory like this
HRESULT WINAPI DllGetActivationFactory(_In_ HSTRING activatableClassId, _Out_ ::IActivationFactory **factory)
{
    return WINRT_GetActivationFactory(activatableClassId, reinterpret_cast<void**>(factory));
}